src/Database/PostgresCompiler.php

```diff
@@ -36,7 +36,9 @@ class PostgresCompiler extends QueryCompiler
     protected bool $_quotedSelectAliases = true;
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
+     *
+     * @var array<string, string>
```

Comment:
This is needed because of a potential bug in psalm. It doesn't seem to inherit the types properly when `@inheritDoc` is used. I have opened an issue for it.